tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "lib": ["es5", "es6"],
     "target": "es5",
     "strict": true,
+    "noImplicitAny": false,
```

Comment:
It had taken a lot of effort to fix all things to enable this and make our code safer. Why we should disable it now?

Reply:
I have removed that line.

Reply:
Oh, my mistake, I thought you change it. Sorry about that.